src/foam/u2/Element.js

```diff
@@ -455,11 +455,19 @@ foam.CLASS({
   ]
 });
 
-// ???: What does this do?
 foam.CLASS({
   package: 'foam.u2',
   name: 'RenderSink',
-  implements: [ 'foam.dao.Sink' ],
+  implements: [
+    'foam.dao.Sink'
+  ],
+
+  documentation: `
+    Given a DAO and a function that maps from an object in that DAO to an
+    Element, apply the function to each object in the DAO and add the returned
+    elements to the view. Will render again every time the DAO is put to or
+    removed from.
```

Comment:
This is worded oddly.

Reply:
Maybe you can try rewording "Given a DAO and a function that maps from an object in that DAO to an Element"